sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md

```diff
@@ -19,6 +19,9 @@
 - Renamed `getValue*` accessor methods on `DocumentField` to `getValueAs*`
 - Renamed `DocTypeInfo` model to `DocumentTypeDetails`
 - Renamed `docTypes` property on `DocumentModelDetails` model to `documentTypes`
+- Renamed `trainingFilesUrl` to `blobContainerUrl` in `beginBuildModel` method
+- Changed method `beginBuildModel(String, DocumentModelBuildMode, BuildModelOptions)` to `beginBuildModel(String, DocumentModelBuildMode, String, BuildModelOptions)`
+- Removed `prefix` property on model `BuildModelOptions`
```

Comment:
From my understanding, Java also has a simpler overload that takes only the document and the build mode, is that right?
```java
beginBuildModel(String, DocumentModelBuildMode)
```

Reply:
Yes, it does. The max overload after this should have the options and prefix